samples/deploy_and_test.py

```diff
@@ -584,17 +586,12 @@ def fetch_ias_token(self, user):
         return self.ias_token
 
     def delete(self):
-        IasAccess.delete(self.ias_service_name)
-
-    @staticmethod
-    def delete(ias_service_name):
-        ias_service_key_name = ias_service_name + '-key'
         logging.info("Deleting service key '{}' for '{}' IAS service"
-                     .format(ias_service_key_name, ias_service_name))
+                     .format(self.ias_service_key_name, self.ias_service_name))
```

Comment:
Hi @liga-oz is it a good practice to use self in a static method? instead we want to use parameter and local variable...